cmd/utils/customflags.go

```diff
@@ -154,11 +154,11 @@ func (b *bigValue) String() string {
 }
 
 func (b *bigValue) Set(s string) error {
-	int, ok := math.ParseBig256(s)
-	if !ok {
+	intVal, ok := math.ParseBig256(s)
+	if !ok || intVal == nil {
```

Comment:
`intVal == nil` check is not needed, right? Ideally, anything that got parsed successfully should return a non-nil value.

Reply:
fixed